wikichron/dash/apps/monowiki/main.py

```diff
@@ -445,10 +378,8 @@ def time_axis(signal, selected_timeaxis, selection_json):
         wikis = selection['wikis']
         metrics = extract_metrics_objs_from_metrics_codes(selection['metrics'])
 
-        data = data_controller.load_and_compute_data(wikis, metrics)
-
         # get time axis of the oldest one and use it as base numbers for the slider:
-        time_axis_index = data_controller.generate_longest_time_axis([ wiki for wiki in data[0] ],
+        time_axis_index = data_controller.generate_longest_time_axis([ metric for metric in metrics ],
```

Comment:
I doubt you need to use this function. Since there's only one wiki selected the time_axis will be always the number of months of that wiki.

Reply:
I think I'll leave it like that for now, as the time_axis is being cached in that function. I'll rename the function though, as you suggest in the latter comment.